### PR TITLE
bring 'propose' back to life

### PR DESCRIPTION
**Pull request by oriharel**

### CODE REVIEW

File: client/view/propose.html [hunk 1]
```diff
@@ -1,3 +1,128 @@
...
+    {{/if}}
+
+    <h2>Timeline</h2>
+    <p>
+        Call for paper ends: <strong>Jan 1</strong>. No kidding.
+    </p>
+    <h2>What are the Topics?</h2>
+    <p>
+        To get a good sense of topics we're interested in you can check out the talks from
+        <a href='http://summit2013.reversim.com/'>last year's conference</a>
```
Comment (rantav) [inline client/view/propose.html]: you can add another link to rs14 here

File: client/view/propose.html [hunk 2]
```diff
@@ -1,3 +1,128 @@
...
+        for local developers and by local developers. There are awesome developers here in Israel.
+        Having said that, if you as a speaker would prefer to speak in English that's totally fine.
+        If you are a non Israeli speaker and would like to present in this conference, you are
+        most welcome, that's perfectly fine, you may do so in English,
+        but just be aware that you'll find yourself bored to death in all other non-english sessions,
+        so bare that in mind of you plan to travel specifically for this event.
+    </p>
+    <h2>Selection Process</h2>
+    <p>
+        We have a wonderful group of <a href='/about'>moderators</a> that would help you
```
Comment (rantav) [inline client/view/propose.html:105]: well, this link is currently broken, but I'll bring it to life...

File: client/view/nav.html [hunk 3]
```diff
@@ -8,7 +8,9 @@
     <a href="{{pathFor 'agenda'}}">Agenda</a>
   </li>
   -->
-  <!--
+    <li class="{{activeClass 'propose'}}"><a href="{{pathFor 'propose'}}"><strong>Propose</strong></a></li>
```
Comment (rantav) [inline client/view/nav.html]: Please don't add it to the navbar just yet, I want to open the CFP in a few days so don't want this to "leak"

**Comment by rantav** [comment]
ok, looks good, did you test it and get it working? 
The next step I guess is to add the proposals and proposal pages (list of proposals and single prop page). 

Anyway - please don't add it to the navbar just yet, I want to open the CFP in a few days so don't want this to "leak"

**Comment by oriharel** [comment]
Ok. I didn't test a full proposal creation - I guess I need to query the db to see if the creation went fine. got it about the nav bar

**Comment by rantav** [comment]
yes, either the db (`meteor mongo`) or the proposals page (which is probably still alive since it works here http://summit2014.reversim.com/proposals)

**Comment by rantav** [comment]
Was already merged in PR #2

**Pull request closed without merging**